aws/logs_monitoring/tools/list_layers.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+# Lists most recent layers ARNs across regions to STDOUT
+# Optionals args: [layer-name] [region]
+
+set -e
+
+LAYER_NAMES=("Datadog-Forwarder")
+AVAILABLE_REGIONS=$(aws ec2 describe-regions | jq -r '.[] | .[] | .RegionName')
+LAYERS_MISSING_REGIONS=()
+
+# Check region arg
+if [ -z "$2" ]; then
+    >&2 echo "Region parameter not specified, running for all available regions."
+    REGIONS=$AVAILABLE_REGIONS
+else
+
+    >&2 echo "Region parameter specified: $2"
+    if [[ ! "$AVAILABLE_REGIONS" == *"$2"* ]]; then
+        >&2 echo "Could not find $2 in available regions:" $AVAILABLE_REGIONS
+        >&2 echo ""
+        >&2 echo "EXITING SCRIPT."
+        exit 1
+    fi
+    REGIONS=($2)
+fi
+
+# Check region arg
```

Comment:
This comment talks about a "region" arg, but I think the code refers to the first arg ("layer-name"). However, there is only one possible layer name so maybe that argument and all of this code can be removed?

Reply:
This file is pretty much copy pasted from the python layer. I think it will make it easier to port changes back if we avoid unnecessary changes. Wondering if it might make sense to abstract our layer tooling into a reusable utility that we share between codebases, and only keep settings like LAYER_NAME in the local repo.

Reply:
Yeah, a reusable layer deploy tool could be a smart idea.

In the mean time, I buy that it's reasonable to leave this extra argument here (I don't think it does any harm). Some of my other comments still apply, I think there's potential for confusion with some of the copy-pasted stuff.